ccd-definition/CaseEventToFields/InvalidHwFReference-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,55 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "INVALID_HWF_REFERENCE",
+    "CaseFieldID": "caseTitleWithHwFReference",
```

Comment:
You need to adapt the title to use the HWF reference number of claim fee or hearing fee depending on the type. You can check other PRs.